src/screens/ProfileScreen/elements/KarmaScore.tsx

```diff
@@ -30,42 +31,44 @@ export const KarmaScore = ({score}: KarmaScoreProps) => {
         </View>
         <Text
           style={{
-            fontWeight: '500',
+            fontFamily: fonts.inter[500],
             fontSize: 24,
             lineHeight: 36,
-            marginRight: 8
+            marginRight: 8,
+            color: COLORS.white2
           }}>{`${score} Karma`}</Text>
         <Tooltip
           allowChildInteraction={true}
           isVisible={isTooltipShown}
           placement={'bottom'}
-          backgroundColor={'rgba(0,0,0,0)'}
           closeOnContentInteraction={false}
           onClose={() => setIsTooltipShown(false)}
           closeOnBackgroundInteraction={true}
           contentStyle={{
             borderRadius: 10,
             padding: 16,
-            height: 'auto'
+            height: 'auto',
+            backgroundColor: COLORS.almostBlack
           }}
           content={
             <View>
               <Text
                 style={{
-                  fontWeight: '600',
+                  fontFamily: fonts.inter[600],
                   fontSize: normalize(16),
                   lineHeight: normalize(24),
-                  marginBottom: normalize(8)
+                  marginBottom: normalize(8),
+                  color: COLORS.white2
                 }}>
                 What is my Karma Score?
               </Text>
               <Text
                 style={{
-                  fontWeight: '400',
+                  fontFamily: fonts.inter[400],
                   fontSize: normalize(12),
                   lineHeight: normalize(18),
                   marginBottom: normalize(8),
-                  color: '#69707C'
+                  color: COLORS.gray500
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The style properties of the Text component within the Tooltip content have been updated. The `fontFamily` property has been added and the `color` property has been changed. These changes seem to be part of an effort to standardize the look and feel of the app. Please ensure that these changes do not conflict with other parts of the application and that they adhere to the design guidelines.

```diff
-                  fontWeight: '600',
-                  color: '#FFFFFF'
+                  fontFamily: fonts.inter[600],
+                  color: COLORS.white2
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->